docs/preview/03-Features/powershell/azure-logic-apps.md

```diff
@@ -293,25 +294,23 @@ Typically done as the last task of the release pipeline, right after the deploym
-| ApiVersion        | no        | The version of the management API to be used.  (default: `2016-06-01`)                                              |
+| Parameter         | Mandatory | Description                                                                                                                                                 |
+| ----------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ResourceGroupName | yes       | The resource group containing the Azure Logic Apps.                                                                                                         |
+| DeployFileName    | no        | If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file.                                         |
+| ResourcePrefix    | no        | In case the Azure Logic Apps all start with the same prefix, you can specify this prefix through this parameter instead of updating the configuration-file. |
+| EnvironmentName   | no        | The name of the Azure environment where the Azure Logic App resides. (default: `AzureCloud`)                                                                |
+| ApiVersion        | no        | The version of the management API to be used.  (default: `2016-06-01`)                                                                                      |
 
 The schema of this configuration file is a JSON structure of an array with the following inputs:
 
 | Node        | Type            | Description                                                                                                           |
 | ----------- | --------------- | --------------------------------------------------------------------------------------------------------------------- |
-| Description | `string`        | Description of Azure Logic App set to enable.                                                                        |
-| CheckType   | `enum`          | _Not taken into account for enabling Logic Apps._ |
-| StopType    | `enum`          | `None`: don't enable to given Logic Apps. |
-|             |                 | `Immediate`: enable the given Logic Apps. |
-| LogicApps   | `string array`  | Set of Logic App names to enable.                                                                                    |
-
+| Description | `string`        | Description of Azure Logic App set to enable.                                                                         |
+| CheckType   | `enum`          | _Not taken into account for enabling Logic Apps._                                                                     |
+| StopType    | `enum`          | `None`: don't enable to given Logic Apps.                                                                             |
```

Comment:
This fix makes sure that it does not take forever to disable Logic Apps, can the same be said for enabling Logic Apps from a configuration file?

Reply:
This is only applicable to where `Get-AzLogicAppRunHistory` is used. I found 2 other scripts so will make a change for that also 👍🏻

Reply:
Super, thx! 👍